### PR TITLE
cmake

### PR DESCRIPTION
**Pull request by sarrow104**

### CODE REVIEW

**Comment by NingLeixueR** [comment]
合并

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,8 @@
 *.exe
 *.out
 *.app
+
```
Comment (NingLeixueR) [inline .gitignore]: 123

**Comment by sarrow104** [comment]
extract example cpp
and create new install stuff

**Comment by NingLeixueR** [comment]
提交

**Pull request merged**